File: visa-dev-samplecode-master/vdp-c-sharp/vdp-c-sharp/VDPApiTest/vta/ManagePortfoliosTest.cs
using Microsoft.VisualStudio.TestTools.UnitTesting;
using System.Configuration;
using System.Collections.Generic;

namespace Vdp
{
    [TestClass]
    public class ManagePortfoliosTest
    {
        private VisaAPIClient visaAPIClient;

        public ManagePortfoliosTest()
        {
            visaAPIClient = new VisaAPIClient();
        }

        [TestMethod]
        public void TestPortfolios()
        {
            string baseUri = "vta/";
            string resourcePath = "v3/communities/" + ConfigurationManager.AppSettings["vtaCommunityCode"] + "/portfolios";
            Dictionary<string, string> headers = new Dictionary<string, string>();
            headers.Add("ServiceId", ConfigurationManager.AppSettings["vtaServiceId"]);
            string status = visaAPIClient.DoMutualAuthCall(baseUri + resourcePath, "GET", "Get Portfolio Details Test", "", headers);
            Assert.AreEqual(status, "OK");
        }
    }
}

File: vdp-php/tests/vta/ManagePortfoliosTest.php
<?php

namespace Vdp;

class ManagePortfoliosTest extends \PHPUnit_Framework_TestCase {
	
	public function setUp() {
		$this->conf = parse_ini_file ( "configuration.ini", true );
		$this->visaAPIClient = new VisaAPIClient;
	}
	
	public function testPortfolios() {
		$baseUrl = "vta/";
		$resourcePath = "v3/communities/".$this->conf['VDP'] ['vtaCommunityCode']."/portfolios";
		$statusCode = $this->visaAPIClient->doMutualAuthCall( 'get', $baseUrl.$resourcePath, 'Get Portfolio Details Test', '', array("ServiceId: ".$this->conf['VDP'] ['vtaServiceId'] ));
		$this->assertEquals($statusCode, "200");
	}
}
